portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Creates Slideshow Carousel to go through my projects*/
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+* @param slideNum: Index of Slide to navigate to
+*/
+
+// Navigate to next slide
+function plusSlides(slideNum) {
+  showSlides("arrow", slideIndex += slideNum);
+
 }
+
+// Navigate to slide indicated by slideNum
```

Comment:
nit: please change the function level comments align with the style guide. e.g. Navigate -> Navigates.
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
done